src/main/java/teammates/common/datatransfer/questions/FeedbackContributionQuestionDetails.java

```diff
@@ -44,6 +44,7 @@ public class FeedbackContributionQuestionDetails extends FeedbackQuestionDetails
 
     private static final Logger log = Logger.getLogger();
 
+    private boolean isZeroSum;
```

Comment:
This field is defined, but I don't see it being used in the backend. Where is the validation that makes use of this field?